src/pps/pps.proto

```diff
@@ -940,6 +940,33 @@ message CreatePipelineTransaction {
   string effective_json = 3; // the effective spec: the result of merging the user JSON into the cluster defaults
 }
 
+message ProjectDefaults {
+  CreatePipelineRequest create_pipeline_request = 1;
+}
+
+message GetProjectDefaultsRequest {
+  pfs_v2.Project project = 1;
+}
+
+message GetProjectDefaultsResponse {
+  // A JSON-encoded ProjectDefaults message, this is the verbatim input passed
+  // to SetProjectDefaults.
+  string project_defaults_json = 1;
```

Comment:
it makes sense that the pipeline spec is passed in verbatim, but does it also make sense for the Defaults object?

Reply:
I believe so — just like with cluster defaults and the pipeline spec, the user experience is that this is JSON.